packages/story-editor/src/components/floatingMenu/elements/loop.js

```diff
@@ -17,22 +17,31 @@
 /**
  * External dependencies
  */
-import { Icons } from '@googleforcreators/design-system';
-import { __ } from '@googleforcreators/i18n';
+import styled from 'styled-components';
 
 /**
  * Internal dependencies
  */
-import { IconButton } from './shared';
+import { useStory } from '../../../app';
+import LoopPanelContent from '../../panels/shared/loopPanelContent';
+import { useProperties } from './shared';
+
+const StyledLoopContent = styled(LoopPanelContent)`
+  gap: 8px;
+`;
 
 function Loop() {
-  return (
-    <IconButton
-      Icon={Icons.ArrowLeftright}
-      title={__('Loop video', 'web-stories')}
-      onClick={() => {}}
-    />
+  const { loop } = useProperties(['loop']);
+  const updateSelectedElements = useStory(
+    (state) => state.actions.updateSelectedElements
   );
+
+  const handleChange = () =>
+    updateSelectedElements({
+      properties: ({ loop: oldLoop }) => ({ loop: !oldLoop }),
+    });
+
+  return <StyledLoopContent loop={loop} onChange={handleChange} />;
```

Comment:
Very clean! The existing component was already perfect (except for minor styling).